backend/tests/controllers/profileController.test.js

```
@@ -0,0 +1,86 @@
+const jwt = require('jsonwebtoken');
+const profileController = require('../../src/controllers/profileController');
+const profileService = require('../../src/services/profileService');
+const { sendSuccessResponse, sendErrorResponse } = require('../../src/utils/responseHelper');
+const { UserDoesNotExistError, ValidationError } = require('../../src/utils/errors');
+
+jest.mock('jsonwebtoken');
+jest.mock('../../src/services/profileService');
+jest.mock('../../src/utils/responseHelper');
+
+describe('profileController.completeRegistration', () => {
+  let req, res;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)